### ISSUE TITLE
LB.1+S:R190S (88 seqs, 25 countries)

### ISSUE THREAD

**Issue by xz-keg** [cov-lineages/pango-designation#2746]
R190S is becoming very convergent with 31del lineages. There's at least 10 each separate emergences of this under LB.1 and KP.2.3, and many emerging under KP.3.1.1.  

From branch 12 of https://github.com/sars-cov-2-variants/lineage-proposals/issues/1771

LB.1+G22132T(S:R190S)

GISAID query:G22132T, G22111T, T3565C,del21653,-4784, -4683, -29095,-18131, -21865, -18395,-14023, -14024,-G19669A,-C320G,-C1454T,-T3679C,-C10340A,-C15386T,-C2143T,-A18504G(may include -s as convergent goes by)
No. of seqs: 21(Australia 4 Denmark 1 India 2 Netherlands 1 Singapore 1 Sweden 1 USA 11(2 from Turkey 2 from Pakistan 1 from UAE)

First: EPI_ISL_19226761, Australia, 2024-6-15
Latest: EPI_ISL_19344335, USA, 2024-8-10

[usher](https://nextstrain.org/fetch/raw.githubusercontent.com/aviczhl2/lineage_proposal_jsons/main/lb_1_r190s.json?label=id:node_5705752)
![image](https://github.com/user-attachments/assets/7b79e959-30ae-4088-8085-b25fa16cfb11)

**Comment by aviczhl2** [comment]
24, Canada

**Comment by aviczhl2** [comment]
27, Japan

**Comment by aviczhl2** [comment]
45, Italy, UK

**Comment by FedeGueli** [comment]
Some more -s:
-G19669A,-C320G,-C1454T,-T3679C

New sequences: G22132T, G22111T, T3565C,del21653,-4784, -4683, -29095,-18131, -21865, -18395,-14023, -14024,-G19669A,-C320G,-C1454T,-T3679C

39 Seqs

**Comment by xz-keg** [comment]
47 now, including a new GBW sample from Iran

**Comment by xz-keg** [comment]
> 11/47 after Sept 1, it is still circulating to some extent. @corneliusroemer has the designation policy changed in some way focusing on just fast lineages and recombinants or is t just lack of time and resources to reduce number of lineages designated? it has to be say there is very little faster than KP.3.1.1 around and further less able to stay close to XEC. I m asking just to know how to set the proposals transferred here.

Maybe he is just lack of time. There has been only one person designating things since early 2023. And there are as much as ~2000 designations during this period.

 That's too much work for a single person, especially considering he is not even full-time working on this, he also have other personal issues to do.

**Comment by FedeGueli** [comment]
> > 11/47 after Sept 1, it is still circulating to some extent. @corneliusroemer has the designation policy changed in some way focusing on just fast lineages and recombinants or is t just lack of time and resources to reduce number of lineages designated? it has to be say there is very little faster than KP.3.1.1 around and further less able to stay close to XEC. I m asking just to know how to set the proposals transferred here.
> 
> Maybe he is just lack of time. There has been only one person designating things since early 2023. And there are as much as ~2000 designations since then. TWO THOUSAND designations!
> 
> That's too much work for a single person, especially considering he is not even full-time working on this, he also have other personal issues to do.

Yeah i know that, just to understand if designation policy changed consequently and we have to tune our proposal intensity to that (here on pango i mean)

**Comment by FedeGueli** [comment]
60

**Comment by xz-keg** [comment]
63, Bahrain, Qatar, Mexico

**Comment by xz-keg** [comment]
68,Germany

**Comment by FedeGueli** [comment]
73 Poland, Oman (old), Canada SK

**Comment by DailyCovidCases** [comment]
Slowing down now but still worth opening

**Comment by Over-There-Is** [comment]
83 South Africa, Luxembourg

**Comment by FedeGueli** [comment]
-s to add to the query: -C10340A,-C15386T,-C2143T,-A18504G

Resulting in:
G22132T, G22111T, T3565C,del21653,-4784, -4683, -29095,-18131, -21865, -18395,-14023, -14024,-G19669A,-C320G,-C1454T,-T3679C,-C10340A,-C15386T,-C2143T,-A18504G

76 Samples

**Comment by FedeGueli** [comment]
79 now with a GBW sample from Morocco in the small 478T sub branch:
<img width="1173" alt="Screenshot 2024-12-14 alle 16 30 33" src="https://github.com/user-attachments/assets/751c214d-06e9-4d15-8090-8371b60b6493" />
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_test_1164e_da2930.json?c=userOrOld&label=id:node_7206794

ping @corneliusroemer this is worth designating even if not super fast:
 6 months around and from Indian Peninsula expanded south to Singapore, Australia and then west touching one after one Middleastern and then southern Mediterranean countries until Morocco in December.

**Comment by xz-keg** [comment]
80, Morocco. 

There seems to be an S:K478T sub-branch of this.

**Comment by xz-keg** [comment]
> 79 now with a GBW sample from Morocco in the small 478T sub branch: <img alt="Screenshot 2024-12-14 alle 16 30 33" width="1173" src="https://private-user-images.githubusercontent.com/87669813/395792238-751c214d-06e9-4d15-8090-8371b60b6493.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ1MTM2MDgsIm5iZiI6MTczNDUxMzMwOCwicGF0aCI6Ii84NzY2OTgxMy8zOTU3OTIyMzgtNzUxYzIxNGQtMDZlOS00ZDE1LTgwOTAtODM3MWI2MGI2NDkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE4VDA5MTUwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwY2FjMDc4ZWE0ZTU0ZTViYTI3ZmY2ODRmOTdjM2YxZmNkNTUxMjRmODJiMGYzMzcwNGRkNDdhMDdhN2VjMGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.I5inVtUN9KV0Pgv89v_9h6Cma0OhUgXgjDsBXecjYN0"> https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/hgPhyloPlace/subtreeAuspice1_genome_test_1164e_da2930.json?c=userOrOld&label=id:node_7206794
> 
> ping @corneliusroemer this is worth designating even if not super fast: 6 months around and from Indian Peninsula expanded south to Singapore, Australia and then west touching one after one Middleastern and then southern Mediterranean countries until Morocco in December.

The Japanese seqs on this branch are from Myanmar.

**Comment by FedeGueli** [comment]
One more for the 478T branch of this from GBW

**Comment by DailyCovidCases** [comment]
@xz-keg Dead please close it

**Comment by xz-keg** [comment]
88,. greece, slovenia

**Comment by FedeGueli** [comment]
> 88,. greece, slovenia

likely dead already

**Comment by Over-There-Is** [comment]
Pakistan +8

**Comment by FedeGueli** [comment]
> Pakistan +8
still old ones

**Comment by FedeGueli** [comment]
Its S:K478T branch apparently recombined with LF.7.3.1 that lost 182R 444R 475V, gaining 183H and 478T from this.
analysis: https://github.com/sars-cov-2-variants/lineage-proposals/issues/2457#issuecomment-2692155206

**Comment by xz-keg** [comment]
designated LB.1.11

**Comment by FedeGueli** [comment]
> One more for the 478T branch of this through GBW

designated LB.1.11.1 the 478T branch

**Comment by FedeGueli** [comment]
> Its S:K478T branch apparently recombined with LF.7.3.1 that lost 182R 444R 475V, gaining 183H and 478T from this. analysis: [sars-cov-2-variants/lineage-proposals#2457 (comment)](https://github.com/sars-cov-2-variants/lineage-proposals/issues/2457#issuecomment-2692155206)

This RECOMBINANT is now desingated XFR

**Comment by FedeGueli** [comment]
> designated LB.1.11

via https://github.com/cov-lineages/pango-designation/commit/1976d3141b99fba901502a0fab3b5c88584014d2
